src/models/window.rs

```diff
@@ -147,9 +148,13 @@ impl Window {
             value = self.normal.w();
         } else if self.floating() && self.floating.is_some() {
             let relative = self.normal + self.floating.unwrap();
-            value = relative.w() - (self.margin * 2) - (self.border * 2);
+            value = relative.w()
+                - (self.margin.clone().left() + self.margin.clone().right())
```

Comment:
@mautamu or @lex148: may I ask, why you apply a margin to a floating window?

Reply:
I’m not sure exactly why it’s applied. Can try running it without and see what happens.

Reply:
Might test myself as well. Was just curious if there was some reasoning behind that.